src/handlers/opensea-api.ts

```diff
@@ -20,17 +18,13 @@ export const apiGetAccountUniqueTokens = async (
   page: any
 ) => {
   try {
-    const API_KEY =
-      network === NetworkTypes.rinkeby
-        ? OPENSEA_RINKEBY_API_KEY
-        : OPENSEA_API_KEY;
     const networkPrefix = network === NetworkTypes.mainnet ? '' : `${network}-`;
     const offset = page * UNIQUE_TOKENS_LIMIT_PER_PAGE;
-    const url = `https://${networkPrefix}api.opensea.io/api/v1/assets`;
+    const url = `https://${networkPrefix}nftp.rainbow.me/api/v1/assets`;
```

Comment:
FYI if we don't want to break compatability downstream then this needs to be configurable too

Reply:
I can add this part of the string to the env if that works: `nftp.rainbow.me/api/v1/assets`

can move the whole thing over once me migrate to only using the v2 api but there's some tradeoffs rn

Reply:
Only the `nftp.rainbow.me` portion needs to be configurable because if downstream changes it to `api.opensea.io` and supply a key via `NFT_API_KEY` then nothing has changed for them

Reply:
should be gucci now